apps/api/src/controllers/competition.controller.ts

```diff
@@ -394,7 +394,14 @@ export function makeCompetitionController(services: ServiceRegistry) {
           queryParams,
         });
 
-        res.status(200).json(result);
+        res.status(200).json({
```

Comment:
Looking at the changes to the competition service, it looks like there are other endpoints that need the pagination response structure preserved: getCompetitionTrades and getAgentTradesInCompetition